acceptance/tests/ensure_puppet-agent_paths.rb

```diff
@@ -118,6 +118,19 @@ def config_options(agent)
   end
 end
 
+step 'test puppet hiera file install' do
+  agents.each do |agent|
+    hiera_config = config_options(agent).select {|v| v[:name] == :hiera_config}.first[:expected]
+    env_path = config_options(agent).select {|v| v[:name] == :environmentpath}.first
+    hiera_env_config = "#{env_path[:expected]}/production/hiera.yaml"
+
+    [hiera_config, hiera_env_config].each do |file|
+      on(agent, "cat '#{file}'") do |cat_result|
+        assert_match(/version: 5/, cat_result.stdout, "Expected hiera version to be set in #{file}")
```

Comment:
Talked with John and we should leave it as "5" instead of a range. As this shouldn't be changing very often and when it does change because we changed the file, then this test should be updated as the file changed.